bookstore-vapor/Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM ubuntu:jammy
```

Comment:
This base image is pretty big. Could we change it with a minimal Linux image ?

Reply:
yes I will look into it

Reply:
I can not find a smaller image that works with the binary, at least not according to the documentation

Reply:
this image is ~30mb big so I don't think that is too bad